frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -598,6 +598,10 @@ describe("scenarios > question > notebook", () => {
         .click();
 
       cy.contains(/^redundant input/i).should("not.exist");
+
+      cy.findAllByRole("button")
+        .contains("Done")
+        .should("not.be.disabled");
```

Comment:
The "correct" syntax for this custom command is:
`cy.findAllByRole("button", { name: "Done" })`

Reply:
Another reason against using `contains`. This doesn't check the button at all. It checks the div.
![image](https://user-images.githubusercontent.com/31325167/109527272-c0f5c900-7ab3-11eb-8f28-66bfd1bdf45b.png)

Reply:
Also, why is there another button check at this line when we had it already just a few lines above?
https://github.com/metabase/metabase/pull/14997/files#diff-7db15743c062b782ece95ce8a874e0510f2c73012d08e1656bd3a0ce1e55c639L595-L598

Reply:
Good points! I've since fixed the issues.